compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToAIE.cpp

```diff
@@ -264,6 +264,12 @@ LogicalResult coreToAIE(IRRewriter &rewriter, AMDAIE::CoreOp coreOp,
   auto walkResult = aieCoreOp.walk([&](Operation *op) {
     rewriter.setInsertionPoint(op);
     if (TypeSwitch<Operation *, LogicalResult>(op)
+            .Case<AMDAIE::LogicalObjectFifoAccessOp>([&](auto accessOp) {
```

Comment:
Temporary to skip `LogicalObjectFifoAccessOp` for now. Will be updated in a follow-up, see https://github.com/nod-ai/iree-amd-aie/compare/jornt_cpp_pipeline?expand=1